robot_smach_states/src/robot_smach_states/manipulation/place.py

```diff
@@ -221,8 +224,14 @@ def __init__(self, robot: Robot, item_to_place: Designator[Entity], place_pose:
                                                 'arrived': 'PUT'})
 
             smach.StateMachine.add('PUT', Put(robot, item_to_place, locking_place_designator, arm),
-                                   transitions={'succeeded': 'done',
-                                                'failed': 'failed'})
+                                   transitions={'succeeded': 'UNLOCK_DESIGNATOR_SUCCESS',
+                                                'failed': 'UNLOCK_DESIGNATOR_FAILED'})
+
+            smach.StateMachine.add('UNLOCK_DESIGNATOR_SUCCESS', UnlockDesignator(locking_place_designator, False),
+                                   transitions={'unlocked': 'done'})
+
+            smach.StateMachine.add('UNLOCK_DESIGNATOR_FAILED', UnlockDesignator(locking_place_designator, False),
+                                   transitions={'unlocked': 'failed'})
```

Comment:
Why does the designator need to be unlocked? If another place is needed wouldnt you need to make a new designator?